test/basic-querying.test.js

```diff
@@ -129,6 +129,16 @@ describe('basic-querying', function() {
 
     before(seed);
 
+    before(function setupDelayingLoadedHook(cb) {
```

Comment:
Do not ignore it, let's do it this way for consistency. Please add this to the style guide @Amir-61.

/cc @bajtos